Cargo.toml

```diff
@@ -13,7 +13,7 @@ browser-panic-hook = "0.2"
 chrono = { version = "0.4.30", default-features = false, features = ["wasmbind"] }
 gloo-utils = "0.2"
 log = "0.4"
-patternfly-yew = { version = "0.5.0-rc.1", features = ["icons-fab", "tree"] }
+patternfly-yew = { git = "https://github.com/patternfly-yew/patternfly-yew", rev = "379c01f", features = ["tree", "dual_list_selector", "icons-fab"] }
```

Comment:
A quick one: I would keep the dependency as is and re-enable the patch down below in the file.

If you want to do that, cool. But I can also do that later.

Reply:
Ah, nevermind. I was too impatient to play with it :)